src/components/common/Extensions/NotificationBanner/types.ts

```diff
@@ -1,20 +1,29 @@
 import { MessageDescriptor } from 'react-intl';
+import { STATUS_TYPES } from '~v5/shared/StatusText/types';
 
 export interface NotificationBannerProps {
-  status: StatusType;
+  status: (typeof STATUS_TYPES)[keyof typeof STATUS_TYPES];
   title?: string | MessageDescriptor;
-  actionText?: string | MessageDescriptor;
-  actionType?: ActionType;
-  redirectUrl?: string;
   isFullSize?: boolean;
   isAlt?: boolean;
-  onClick?: () => void;
+  action?:
+    | {
+        type: 'copy';
+        copyContent: string;
+        actionText: string | MessageDescriptor;
```

Comment:
Instead of allowing `string | MessageDescriptor` just do `React.ReactNode` and you will be able to pass for example `<FormattedMessage id="..." />` component

Reply:
Changed to React.ReactNode